docs/codeql/codeql-cli/query-reference-files.rst

```diff
@@ -25,22 +25,24 @@ Defining a query reference file
 
 Each query reference file, ``.qlref``, contains a single line that defines
 where to find one query. The location must be defined relative
-to the root of the QL pack that contains the query. 
-Usually, this is a QL pack specified by the ``libraryPathDependencies`` for the test pack.
+to the root of the CodeQL pack that contains the query.
+Usually, this is either the CodeQL pack that contains the ``.qlref``, a CodeQL pack
+specified in the ``dependencies`` block for the test pack, or a transitive dependency of
+the CodeQL pack.
 
 You should use forward slashes in the path on all operating
-systems to ensure compatibility between systems. 
+systems to ensure compatibility between systems.
 
 Example
-^^^^^^^^
+^^^^^^^
 
 A query reference file to test a JavaScript alert query:
 `DeadAngularJSEventListener.qlref <https://github.com/github/codeql/blob/main/javascript/ql/test/query-tests/AngularJS/DeadAngularJSEventListener/DeadAngularJSEventListener.qlref>`__
 
-The `QL pack <https://github.com/github/codeql/blob/main/javascript/ql/test/qlpack.yml>`__ 
-for the ``javascript/ql/test`` directory defines the ``codeql-javascript`` queries as
+The `CodeQL pack <https://github.com/github/codeql/blob/main/javascript/ql/test/qlpack.yml>`__
+for the CodeQL pack at ``javascript/ql/test`` defines ``codeql/javascript-queries`` as
```

Comment:
> The CodeQL pack for the CodeQL pack

Reads a little weirdly. Maybe this should be:

```suggestion
The ```qlpack.yml`` file <https://github.com/github/codeql/blob/main/javascript/ql/test/qlpack.yml>`__
for the CodeQL pack at ``javascript/ql/test`` defines ``codeql/javascript-queries`` as
```